packages/service-library/src/servicelib/aiohttp/tracing.py

```diff
@@ -72,9 +73,8 @@ def setup_tracing(
     resource = Resource(attributes={"service.name": service_name})
     trace.set_tracer_provider(TracerProvider(resource=resource))
     tracer_provider: trace.TracerProvider = trace.get_tracer_provider()
-    tracing_destination: str = (
-        f"{opentelemetry_collector_endpoint}:{opentelemetry_collector_port}/v1/traces"
-    )
+
+    tracing_destination: str = f"{URL(opentelemetry_collector_endpoint).with_port(opentelemetry_collector_port).with_path('/v1/traces')}"
```

Comment:
are you sure you need to put / in front of v1? well I guess it handles it.

Reply:
Yes, because there is a port. The problem was before.